src/server/handlers/webhook.js

```diff
@@ -0,0 +1,9 @@
+const logging = require('../logging/').default;
+const logger = logging.getLogger('express');
+
+export const notify = (req, res) => {
+  // Acknowledge webhook
+  res.status(200).send();
```

Comment:
Acknowledge, but do nothing.

Handling push events is outside of the scope of this PR; need to find out from @cjwatson how the payload should be handled.